guides/studio-development.md

```diff
@@ -0,0 +1,11 @@
+# Studio Development
+
+In production, the code used to facilitate Studio functionality will be retrieved from the Cloud. However, in order to develop locally, developers will:
+
+- Clone the `cypress-services` repo
```

Comment:
```suggestion
- Clone the `cypress-services` repo (this requires that you be a member of the Cypress organization)
```

Reply:
Good call. Added this to the protocol guide as well.